modules/delegates.js

```diff
@@ -492,16 +571,18 @@ __private.loadDelegates = function(cb) {
 			let secret;
 			try {
 				secret = __private.decryptSecret(
-					encryptedItem.encryptedSecret,
+					encryptedItem,
```

Comment:
That will be more readable code if you pass directly `encryptedItem.encryptedSecret` and accept a direct param instead  of hash in method.

Reply:
> accept a direct param instead of hash in the method

I don't understand this suggestion.

Reply:
You are passing an object `encryptedItem ` and function definition is extracting the required attribute `encryptedSecret` from that object. Instead, I think if we pass a direct attribute `encryptedItem.encryptedSecret` to function call that will be more readable.

Reply:
Ah by `hash` you mean an object?

Reply:
Yes I mean object. Sorry being Ruby addictive thoughts, I tend to use `hash` for key-value paired objects.